### ISSUE TITLE
Tracking for Surface pen

### ISSUE THREAD

**Issue by TacticalTechJay** [ppy/osu#17514]
### Type

Game behaviour

### Bug description

Tracking for Surface pens have been stuttery since using Osu!lazer. Compared to stable, I was able to hover with the pen and tap. Now, I can only tap and am unable to hover.

### Version

2022.327.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/8365688/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8365689/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8365691/updater.log)
[database.log](https://github.com/ppy/osu/files/8365692/database.log)
[network.log](https://github.com/ppy/osu/files/8365693/network.log)

**Comment by bdach** [comment]
i don't know what "stuttery" means. please attach a video, preferably with fps counter showing.

also possibly related to https://github.com/ppy/osu/discussions/16840 since surface device has been mentioned.

**Comment by TacticalTechJay** [comment]
After looking through the linked issue, I tried turning _on_ high precision mouse and changing mouse sensitivity. Turns out having mouse sensitivity set to anything other than 1.0 while having high precision mouse on would give me a different sized area where I can tap with pen or finger. Having it set to 1.0 made the position of the mouse match where I tapped. Disabling high precision mouse actually caused the issue with this stuttery pen.
